### PR TITLE
Add bundlesize.

### PR DESCRIPTION
**Pull request by jbucaran**
Add bundlesize.

### CODE REVIEW

**Comment by jbucaran** [comment]
Hi @siddharthkp 👋😁,

Does bundlesize compare the size of the minified+gzip'd file or only minified?

```diff
+  },
 +  "bundlesize": [
 +    {
 +      "path": "./dist/hyperapp.js",
 +      "threshold": "1.5Kb"
 +    }
 +  ]
```

File: package.json [hunk 1]
```diff
@@ -36,5 +36,11 @@
     "prettier": "~1.4.1",
     "rollup": "^0.41.6",
     "rollup-plugin-uglify": "^2.0.1"
-  }
+  },
+  "bundlesize": [
+    {
+      "path": "./dist/hyperapp.js",
+      "threshold": "1.5Kb"
```
Comment (Swizz) [inline package.json]: `1.5kb` ? Removing router didnt remove the overhead ?

Comment (jbucaran) [inline package.json]: @swizz What do you mean?

**Comment by jbucaran** [comment]
I made this PR an image of [this one here](https://github.com/developit/preact/commit/8a729f99e50ec872410cacf7678aa3572ae0c8bb) in preact. I am not sure I got it right, though, since preact.min.js is `8.1 KB`.

**Comment by Swizz** [comment]
Looking at the source code : [files.js#L12](https://github.com/siddharthkp/bundlesize/blob/master/src/files.js#L12)
```jsx
const size = gzip.sync(fs.readFileSync(path, 'utf8'))
```

**Comment by jbucaran** [comment]
@Swizz Then we're good. Without the router, hyperapp is now 1400 bytes or so. I think 1.5 is a good compromise. I have a feeling that will go up to 1.8 or 2 at some point, but that's okay.

File: package.json (hunk 1 above)
Comment (jbucaran) [inline package.json]: Ah gotcha. I left you a comment in the main thread! 😉

**Comment by zaceno** [comment]
@jbcaran srsly youd be ok w 2kb? Wow man, you've grown  ;)

**Comment by jbucaran** [comment]
@zaceno Under 2kb! Still, I'll aim at 1.4~1.5 max.

**Comment by Swizz** [comment]
@jbucaran Did you miss bundlesize as dependency ?

**Pull request merged**